stonesoup/sensor/radar/radar.py

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
 import copy
-from math import erfc
 from typing import Tuple, Set, Union
 
 import numpy as np
 import scipy.constants as const
+from math import erfc
```

Comment:
This is standard library

Reply:
The line `from typing import Tuple, Set, Union`?

Reply:
`math` is standard library, so should be in first set of imports, where it was before.